addons/mail/static/src/models/call_invite_request_popup_list.js

```diff
@@ -0,0 +1,14 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { one } from '@mail/model/model_field';
+
+registerModel({
+    name: 'CallInviteRequestPopupList',
+    fields: {
+        owner: one('CallSystrayMenu', {
```

Comment:
I'm really not fan of just having `owner` as name, I think it will become maintenance nightmare very quickly.

- if there is a need to add another owner to an existing model, the existing owner field will need to be renamed to differentiate it from the new one, even this field itself didn't change. It doesn't look acceptable to me that we have to rename a field if its purpose and meaning didn't change.
- if the owner type is changing, the name will stay the same, leading to not necessarily paying attention where/how it is used in the code (as there is no need to rename it), but all field access of the owner would need to be changed. This is the inverse of the previous concern: it doesn't look acceptable that a field is not renamed if its meaning actually changed.
- tooling issue, but still relevant: name is too short and finding proper references to owner in code becomes more and more complex as we add them

Reply:
Ok for current compromise to name it more specifically, due to essentially tooling limitations.
Something like `callSystrayMenuOwner`. It's less readable but a bit more maintainable, so a reasonable trade-off.

When it's no longer a maintenance issue, a short name like `owner` is totally fine.